src/random.ts

```diff
@@ -259,8 +259,17 @@ export class Random {
    *
    * @example
    * faker.random.word() // 'Seamless'
+   *
+   * @deprecated
    */
   word(): string {
+    deprecated({
+      deprecated: 'faker.random.word()',
+      similar: 'faker.word.any()',
+      since: 'v6.1.0',
```

Comment:
The upcoming release is v6.2.0
```suggestion
      since: 'v6.2.0',
```

Reply:
That's really old :laughing: